### PR TITLE
Fix and document the verifier

### LINKED ISSUE #231: Verifier undocumented
Vote auditing is not explained, it needs links and documentation on how to do it. The current state makes it completely useless in practical terms.

At least the [README](https://github.com/decidim/decidim-bulletin-board/blob/develop/verifier/README.md) should have instructions (working in several operating systems) on how to verify a vote.

- [ ] Finish PR #192 
- [ ] Add instructions
- [ ] Verify existence of tests

### PR DESCRIPTION
**Pull request by microstudi**
Fixes the verifier for ballots or election logs (a tar file).

- Verifying a single ballot requires python and the library electionguard to be installed.
- Verifying a tar election file requires java and to compile a JAR file.

I've added instructions in the README file to documentat all of this, is not ideal but it can save you a lot of time.

fixes #231 
supersedes #192 
requires #310 to be merged to fix the rubocop issues

image

### CODE REVIEW

File: verifier/README.md [hunk 1]
```diff
@@ -1,13 +1,76 @@
...
+
+```bash
+bin/verify_election ELECTION_TAR_FILE
+```
+- The `ELECTION_TAR_FILE` is the TAR file downloaded from the Decidim application, just after the election is finished and the results published (a file ending in `.tar`). Look for the "Election log" link to find it.
+
+For help and examples, run the script with the `--help` modifier:
+
+```bash
+node src/index.js --help
```
Comment (andreslucena) [inline verifier/README.md]: I think this doesn't work:
```console
$ node src/index.js --help
[Error: ENOENT: no such file or directory, access '--help'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'access',
  path: '--help'
}
```

**Review by andreslucena — CHANGES_REQUESTED** [review]
I just checked the installation instructions. For the moment it seems much better than what we have. There's just a minor detail to fix and we can merge this

File: verifier/README.md (hunk 1 above)
Comment (microstudi) [reply verifier/README.md]: Because it needed to be run inside the "verify" folder.
I've update the instructions

**Review by andreslucena — APPROVED** [review]

File: verifier/README.md (hunk 1 above)
Comment (andreslucena) [reply verifier/README.md]: Confirmed that it was fixed

**Pull request merged**